tests/models/llama/test_modeling_llama.py

```diff
@@ -375,6 +376,41 @@ def test_model_rope_scaling(self, scaling_type):
         # The output should be different for long inputs
         self.assertFalse(torch.allclose(original_long_output, scaled_long_output, atol=1e-5))
 
+    @require_flash_attn
+    @require_torch_gpu
+    @mark.flash_attn_test
```

Comment:
Do you need the `mark` here or was it just for testing?

Reply:
Indeed only for testing purpose, I can spin up flash attn tests quickly as such:
```bash
pytest -m flash_attn_test tests/models/llama/test_modeling_llama.py
```
And quickly iterate

Reply:
Maybe we could eventually slightly rework the `@require...` flags to behave as `@mark` decorators. I'd rather we not define similar decorators twice in tests for readability purposes, and it could be quite useful to have every `@require` offer the possibility to isolate it within our tests.

That's not for this PR however, we can let it like that for now if you want

(cc @ydshieh @ArthurZucker)